src/core/pipelines/manifest/create_manifest_loader.ts

```diff
@@ -33,8 +33,8 @@ import {
   ITransportManifestPipeline,
 } from "../../../transports";
 import tryCatch$ from "../../../utils/rx-try_catch";
-import backoff from "../utils/backoff";
 import errorSelector from "../utils/error_selector";
+import { tryRequestObservableWithBackoff } from "../utils/try_urls_with_backoff";
```

Comment:
Try URLS with backoff ?

Reply:
The issue here is that we do not know on which URL the request will be done.

A ManifestLoader here will just call `scheduleRequest`, give it an opaque observable and expect it to have the same guarantee than a regular request.

We need a refacto somewhere if we want to also allow `scheduleRequest` to give several URLs for the same resources.